README.md

```diff
@@ -8,24 +8,36 @@
 ```
 pip install -r requirements.txt -r requirements-dev.txt
 ```
-
 To run app with Postgres db install additional dependencies:
 ```
 pip install -r requirements-postgres.txt
 ```
-
+#### Build docker image
+To build image using SQLite DB
+```
+docker build -f docker/Dockerfile -t rfhub-new .
+```
+To build image using PostgreSQL DB
+```
+docker build -f docker/Dockerfile-postgres -t rfhub-new-postgres .
+```
 #### Run application (web server)
-
 To run with default (SQLite) database
 ```
 python -m rfhub2
 ```
-
 To run with Postgres database
 ```
 RFHUB_DB_URI=postgresql://postgres:postgres@localhost:5432/postgres python -m rfhub2
 ```
-
+To run application using docker image with default (SQLite) database
+```
+docker run -it -p 8000:8000 rfhub-new
+```
+To run application using docker image with Postgres database
+```
+docker run -it -p 8000:8000 -e RFHUB_DB_URI="postgresql://postgres:postgres@localhost:5432/postgres" rfhub-new-postgres
```

Comment:
This command may be misleading because it expects postgres to be available via localhost interface of the rfhub container. It can be changed either by switching localhost to docker network ip of postgres container or to run rfhub with `--network=host` option. I would go with the second option here, since it can be copied and run without any adjustments.